source/Playnite/Localization/LocSource.xaml

```diff
@@ -777,6 +781,11 @@ Note that the Account Name option only works for accounts whose game libraries h
     <sys:String x:Key="LOCSteamCustomUrlName">Account URL Name</sys:String>
     <sys:String x:Key="LOCSteamLoggedInUser">Authenticated User</sys:String>
     <sys:String x:Key="LOCSteamRequiresApiKey">Account is private, API Key is required.</sys:String>
+    <sys:String x:Key="LOCSteamLinksCommunityHub">Community Hub</sys:String>
+    <sys:String x:Key="LOCSteamLinksDiscussions">Discussions</sys:String>
+    <sys:String x:Key="LOCSteamLinksNews">News</sys:String>

```

Comment:
I think News and Achievements can be general strings as well, those terms do apply to other stores as well.

Reply:
Also if it's possible and you have time, it'd be great to look at #1479 too, since I think they are related since it would help to have Playnite entirely in one language (at least when using steam games). And thanks for the help in this issue